plugins/inputs/t128_metrics/t128_metrics.go

```diff
@@ -173,6 +182,12 @@ func (plugin *T128Metrics) retrieveMetrics(index int, acc telegraf.Accumulator,
 			message = []byte("")
 		}
 
+		if strings.Contains(string(message), "No configured endpoints satisfy the request") {
```

Comment:
with this change will the metrics be ignored until the next time telegraf is restarted, correct? Once legitimate scenario I can think of is for custom metrics and some app wants to monitor it - think plugins. In that case as long as there is a mechanism for telegraf to pick up on the new metrics it will be fine - just curious for documentation and tracking sake is all

Reply:
from my testing, the metrics are ignored until telegraf is restarted

Reply:
i wasn't sure if/how the telegraf docs should be updated in this case